Day-01/java/zFlxw/Solution.java

```diff
@@ -0,0 +1,50 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+public class Main {
```

Comment:
The name of the class should match the name of the file, otherwise this can't be compiled.

Reply:
I've changed the class name